test/TestConfig/scripts/testKitGen/makeGenTool/mkgen.pl

```diff
@@ -318,12 +328,13 @@ sub handle_start {
 
 sub handle_end {
 	my ($p, $elt) = @_;
+	my $neglectedFlag;
```

Comment:
Move initialization of `neglectedFlag` to the file scope and use it to replace `$parseTest->{'disabled'}` in line 455.